src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -440,45 +462,36 @@ JSON example:
 ```JSON
 {
   "PhaseEncodingDirection": "j-",
-  "TotalReadoutTime": 0.095
+  "TotalReadoutTime": 0.095,
+  "B0FieldSource": ["phasediff_fmap0", "pepolar_fmap0"]
```

Comment:
This should be a relative-ish path with the same format as `IntendedFor`, right? Can you replace these with realistic filenames?

Reply:
I misunderstood this proposal. Never mind!